src/v/cloud_topics/level_zero/frontend_reader/reader.h

```diff
@@ -113,8 +113,12 @@ class level_zero_log_reader_impl : public model::record_batch_reader::impl {
     };
 
     // Data from the local log that is not yet hydrated from data in L0
+    //
+    // The data stored in this buffer is ascending order by offset.
```

Comment:
nit: is there also an expected ordering between _unhydrated and _hydrated?

Reply:
yeah that's correct (if non empty)

Reply:
Updated to add this bit

Reply:
Thank yoU!